src/slide.jsx

```diff
@@ -33,10 +36,12 @@ const Slide = React.createClass({
   setZoom() {
     const content = React.findDOMNode(this.refs.content);
     const zoom = (content.offsetWidth / config.width);
-    const contentScale = (content.parentNode.offsetHeight / config.height);
+    const contentScaleY = (content.parentNode.offsetHeight / config.height);
+    const contentScaleX = (content.parentNode.offsetWidth / config.width);
```

Comment:
In the current presenter mode, on desktop at least, it looks ok. Mobile is questionable, but generally useable. I see the difficulty for overview though, because you can't really flexbox that layout.